core/src/main/scala/slamdata/domain/PullRequestDraft.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package precog.domain
+
+import github4s.domain.{PullRequest, PullRequestBase, User}
+
+case class PullRequestDraft(id: Int,
+                            number: Int,
+                            node_id: String,
+                            state: String,
+                            title: String,
+                            body: Option[String],
+                            locked: Boolean,
+                            draft: Boolean,
+                            html_url: String,
+                            created_at: String,
+                            updated_at: Option[String],
+                            closed_at: Option[String],
+                            merged_at: Option[String],
+                            merge_commit_sha: Option[String],
+                            base: Option[PullRequestBase],
+                            head: Option[PullRequestBase],
+                            user: Option[User],
+                            assignee: Option[User]
+                           )
```

Comment:
Another random style nit: we usually don't favor dangling `)` in this fashion. Instead, we put it at the end of the previous line. This distinguishes the scope a bit more from `}` (which we *do* push to the next line!), where the newline inference semantics are different.